articles/install-guide/qjupyter.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Develop with Q# Jupyter notebooks
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.jupyter
+---
+
+# Develop with Q# Jupyter notebooks
+
+Install the QDK for developing Q# operations on Q# Jupyter Notebooks.
+
+Jupyter Notebooks allow in-place code execution alongside instructions, notes, and other content. This environment is ideal for writing Q# code with embedded explanations or quantum computing interactive tutorials. Here's what you need to do to start creating your own Q# notebooks.
+
+IQ# (pronounced i-q-sharp) is an extension primarily used by Jupyter and Python to the .NET Core SDK that provides the core functionality for compiling and simulating Q# operations.
+
+> [!WARNING]
```

Comment:
```suggestion
> [!WARNING]
```
```suggestion
> [!NOTE]
```

Changing to Note instead of Warning because this functionality is expected and doesn't necessarily defy user expectations